build.sbt

```diff
@@ -1,8 +1,18 @@
-pgpPassphrase := sys.env.get("PGP_PASSPHRASE").map(_.toCharArray)
+addCommandAlias(
+  "ci-test",
+  "+scalafmtCheck; +scalafmtSbtCheck; project-docs/mdoc; +coverage; +test; +coverageReport; +coverageAggregate"
+)
+addCommandAlias("ci-docs", "project-docs/mdoc; headerCreateAll")
 
 lazy val pbdirect = project
   .in(file("."))
   .settings(name := "pbdirect")
 
-addCommandAlias("ci-test", "scalafmtCheck; scalafmtSbtCheck; +test")
-addCommandAlias("ci-docs", "compile")
+lazy val `project-docs` = (project in file(".docs"))
+  .aggregate(pbdirect)
```

Comment:
Why does the docs project aggregate the main project?

Reply:
Probably because of copy&paste. Fixing...